lib/beaker-pe/pe-client-tools/install_helper.rb

```diff
@@ -0,0 +1,74 @@
+module Beaker
+  module DSL
+    module InstallUtils
+      module PEClientTools
+
+        def install_pe_client_tools_on(hosts, opts = {})
+          # FIXME: accomodate production released location(s)
+          #{{{
+          product = 'pe-client-tools'
+          required_keys = [:puppet_collection, :pe_client_tools_sha, :pe_client_tools_version]
+
+          unless required_keys.all? { |opt| opts.keys.include?(opt) }
+            raise ArgumentError, "The keys #{required_keys.to_s} are required in the opts hash"
+          end
+          urls = { :dev_builds_url   => "http://builds.delivery.puppetlabs.net",
+          }
+
+          opts = urls.merge(opts)
+
+          block_on hosts do |host|
+            package_name = nil
+            variant, version, arch, codename = host['platform'].to_array
+            case host['platform']
+            when /el-|fedora|sles|centos|cisco_/
+              release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/#{variant}/#{version}/#{opts[:puppet_collection]}/#{arch}"
+              package_name = product.dup
+              package_name << "-#{opts[:pe_client_tools_version]}-1.#{variant}#{version}.#{arch}.rpm" if opts[:pe_client_tools_version]
+            when /debian|ubuntu|cumulus|huaweios/
+              release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/deb/#{codename}/#{opts[:puppet_collection]}"
+              package_name = product.dup
+              package_name << "_#{opts[:pe_client_tools_version]}-1#{host['platform'].codename}_#{arch}.deb" if opts[:pe_client_tools_version]
+            when /windows/
+              release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/#{variant}/#{opts[:puppet_collection]}/x#{arch}"
+              package_name = product.dup
+              package_name << "-#{opts[:pe_client_tools_version]}.1-x#{arch}_VANAGON.msi" if opts[:pe_client_tools_version]
```

Comment:
This will need to change when https://github.com/puppetlabs/pe-client-tools-vanagon/pull/88 gets merged, because it will remove the `VANAGON` suffix from the filename.